setup.py

```diff
@@ -120,10 +120,20 @@ def gen_data_files(*dirs):
     Thanks: http://stackoverflow.com/a/7288382/405682
     """
     results = []
-
+    
+    filter_illegal_extensions = lambda f: os.path.splitext(f)[1] != ".pyc"
+    
     for src_dir in dirs:
         for root, dirs, files in os.walk(src_dir):
-            results.append((root, map(lambda f: os.path.join(root, f), files)))
+            results.append(
+                (
```

Comment:
Yes, unfortunately not an ideal scenario that it's solving :( Basically, it has to list tuples containing all files (listed recursively) in all src_dirs, but the files have to filter so there are no pyc files... if the problem is ugly, the solution is, too.